components/infobox/commons/infobox_league.lua

```diff
@@ -416,7 +417,7 @@ end
 ---@param args table
 ---@return boolean
 function League:liquipediaTierHighlighted(args)
-	return false
+	return Logic.readBool(args.publisherpremier)
```

Comment:
Would it not be better to use the parsed data?

Reply:
is that just replacing `args.publisherpremier` with `self.data.publishertier` which is added through the customParse section?